src/components/ec-btn-dropdown/ec-btn-dropdown.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <div
+    ref="popperReference"
+    class="ec-btn-dropdown"
+    :data-test="$attrs['data-test'] ? `${$attrs['data-test']} ec-btn-dropdown` : 'ec-btn-dropdown'"
+  >
+    <ec-btn
+      :is-disabled="isDisabled"
+      is-reverse
+      category="primary"
+      is-rounded
+      class="ec-btn-dropdown__btn"
+      data-test="ec-btn-dropdown__btn"
+      @click="$emit('click')"
+    >
+      {{ textButton }}
+    </ec-btn>
+    <ec-dropdown-search
+      :items="itemsDropdown"
+      :is-search-enabled="false"
+      :popper-modifiers="popperModifiers"
+      :disabled="isDisabled"
+      @change="(value) => $emit('change', value)"
+      @open="isActive = true"
+      @close="isActive = false"
+    >
+      <ec-btn
+        :is-disabled="isDisabled"
+        is-reverse
+        is-rounded
+        category="primary"
+        :class="{
+          'ec-btn-dropdown__dropdown-btn': true,
+          'ec-btn-dropdown__dropdown-btn--active': isActive,
+        }"
+        data-test="ec-btn-dropdown__dropdown-btn"
+        icon="simple-arrow-drop-down"
+      />
+    </ec-dropdown-search>
+  </div>
+</template>
+
+<script>
+import EcBtn from '../ec-btn';
+import EcDropdownSearch from '../ec-dropdown-search';
+
+export default {
+  name: 'EcBtnDropdown',
+  components: {
+    EcBtn,
+    EcDropdownSearch,
+  },
+  props: {
+    itemsDropdown: {
```

Comment:
I'd call this items. We should not use names of the internal components as prop names.